index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Module 4 Solution Starter</title>
+  <script>
```

Comment:
Please move script tags to right before the body closing tag.

Reply:
Done thank you